file/file-api/src/main/java/uk/gov/justice/services/file/api/requester/StreamingFileRequester.java

```diff
@@ -0,0 +1,15 @@
+package uk.gov.justice.services.file.api.requester;
+
+import java.io.InputStream;
+import java.util.Optional;
+
+public interface StreamingFileRequester extends FileRequester{
+    /**
+     * Requests a file from the FileService as an InputStream.
+     * @param fileId - the unique id of the file.
+     * @param fileMimeType - mime-type of the file.
+     * @param fileName - name of the file.
+     * @return content of the file as InputStream.
+     */
+    Optional<InputStream> requestStreamed(final String fileId, final String fileMimeType, final String fileName);
+}
```

Comment:
We can't name them request as the FileRequester also has method called request, so the classes implementing both interfaces would have issue